core/src/main/java/io/confluent/kafka/schemaregistry/rest/SchemaRegistryConfig.java

```diff
@@ -34,9 +35,6 @@
   // TODO: change this to "http://0.0.0.0:8081" when PORT_CONFIG is deleted.
   private static final String SCHEMAREGISTRY_LISTENERS_DEFAULT = "";
 
-  public static final String KAFKASTORE_SECURITY_PROTOCOL_SSL = "SSL";
```

Comment:
Yeah these were introduced in `3.0.0`. Man, I didn't even think about public constants being breaking changes -- you're right, and good catch. I won't delete them and instead mark them as deprecated.